packages/ide-extension/src/enhancement/enhancements.json

```diff
@@ -9,7 +9,8 @@
                 "exec": {
                     "extensionId": "sapse.sap-ux-application-modeler-extension",
                     "commandId": "sap.ux.environmentcheck.archiveProject"
-                }
+                },
+                "platforms": ["VSCODE", "BAS"]
```

Comment:
Please check from `BAS` to `SBAS`, so that it represents correctly SAP Business Application Studio

Reply:
done!